spec/creditsafe/client_spec.rb

```diff
@@ -47,18 +47,11 @@
         stub_request(:post, URL).to_timeout
       end
 
-      it 'raises an HttpError' do
-        expect { method_call }.to(
-          raise_error(Creditsafe::HttpError)
-        )
-      end
-
       it 'gives a useful error message' do
-        begin
-          method_call
-        rescue Creditsafe::HttpError => err
-          expect(err.message).to include 'Excon::Errors::Timeout'
-        end
+        expect { method_call }.to raise_error(
+          Creditsafe::HttpError,
+          /Excon::Error(?:s)?::Timeout/
```

Comment:
It used to throw an `Excon::Errors::Timeout` but now throws an `Excon::Error::Timeout`.